### PR TITLE
Add various design improvements

### LINKED ISSUE #237: Use "object.og_image" method for OG image
Currently auto-rendered product pages use product image directly for OG image. As the product image is returned with media.voog.com URL, it is blocked on some social media platforms. Recently added `object.og_image` method fixes this issue.

### LINKED ISSUE #238: Add titles to content area tags
Add titles and tooltips to content area tags to help differentiate content areas.

### PR DESCRIPTION
**Pull request by PeeterPaal**
- Use `object.og_image` method for OG image
- Add titles & tooltips to various content areas

Closes #237 
Closes #238

### CODE REVIEW

File: components/layout-footer.tpl [hunk 1]
```diff
@@ -10,12 +10,14 @@
...
+    {%- assign footer_content_title = "footer" | lce -%}
+    {%- assign footer_content_title_tooltip = "content_tooltip_all_pages_same_language" | lce -%}
     <div class="layout-footer__section layout-footer__section--wide layout-footer__section--body">
-      <div class="content-area">{% xcontent name="footer" %}</div>
+      <div class="content-area">{% xcontent name="footer" title=footer_content_title title_tooltip=footer_content_title_tooltip %}</div>
     </div>
 
     <div class="layout-footer__section layout-footer__section--narrow layout-footer__section--social-media">
-      <div class="content-area">{% xcontent name="footer_2" %}</div>
+      <div class="content-area">{% xcontent name="footer_2" title=footer_content_title title_tooltip=footer_content_title_tooltip %}</div>
```
Comment (tanelj) [inline components/layout-footer.tpl:20]: Its too narrow text area. Let's keep only `title_tooltip`
```suggestion
      <div class="content-area">{% xcontent name="footer_2" title_tooltip=footer_content_title_tooltip %}</div>
```

**Review by tanelj — CHANGES_REQUESTED** [review]
Nice, but some improvements:
* "Blog article" `additional_body` use additional_content and content_tooltip_additional_information

**Review by tanelj — APPROVED** [review]
Nice!

File: components/blog-article.tpl [hunk 2]
```diff
@@ -16,9 +16,9 @@
...
       {% assign article_date_format = "long" %}
     {% endif %}
-    
+
     {% if editmode or show_article_date != false %}
-    <div>  
+    <div>
       <time class="post-date formatted cfx{% if article_data_show_date_defined != true %} site-data{% endif %}{% if show_article_date == false %} hide-article-date{% endif %}" datetime="{{ article.created_at | date: '%Y-%m-%d' }}">{{ article.created_at | format_date: article_date_format }}</time>
     </div>
     {% endif %}
```
Comment (pxska) [inline components/blog-article.tpl:24]: Indentation is wrong here:
```suggestion
      <div>
        <time class="post-date formatted cfx{% if article_data_show_date_defined != true %} site-data{% endif %}{% if show_article_date == false %} hide-article-date{% endif %}" datetime="{{ article.created_at | date: '%Y-%m-%d' }}">{{ article.created_at | format_date: article_date_format }}</time>
      </div>
    {% endif %}
```

**Review by pxska — APPROVED** [review]
A small comment + check for long lines. Otherwise good!

**Review by robertpih — APPROVED** [review]

**Pull request merged**